### DISCUSSION TITLE
Feature Request: Add Mediawiki migration functionality

### DISCUSSION THREAD

**Discussion by bmn001** [docmost/docmost#1639] [Ideas]
I've got a mediawiki install with about 300 articles.

I want to be able to import them all - complete with images - to Docmost.

I can sort out the hierarchy after, that's no problem.

Right now it looks like I'd have to convert the articles to markdown, import each article individually, and then upload each image individually to Docmost.

I would like there to be some automated way to do this within Docmost.  It would be a great, low-friction way to get people to adopt Docmost if there was a way to do this from not just Mediawiki but from other document sources.

**Comment by vector090** [comment]
+1 to this

**Reply by silverSl1DE** [reply]
+1
